data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1611 @@
+	on complete
+		payment 250000
+		conversation
+			`You have barely cleared the security perimeter when your commlink chimes twice with a notification of a logistics credit of <payment> being deposited to your account, closely following a message to land on pad 3E5 near the research hanger Taely has been working in.`
+
+
+
+mission "Remnant: Cognizance 23"
+	name "First Flight"
+	description "Escort the Merganser to <stopover>."
+	landing
+	source
+		government "Remnant"
+	waypoint "Farinus"
+	to offer
+		has "Remnant: Cognizance 22: done"
+	on offer
+		event "Remnant: Merganser"
+		conversation
+			`You disembark from the <ship> just in time to watch a trio of small ships taxi out of the hangar where Taely has been working. They are strange little ships, only somewhat larger than the terns and petrels you see on a regular basis. What stands out the most, however is the fact that the three ships seem to be walking of their own accord, moving forward on flexible landing legs. Taely follows them out, and when they reach the landing pad she whistles, and the three ships turn around and cluster around her. She smiles and reaches up to pat them on the nose. The trio sparkle with iridescent ripples as they move around, the skin flexing and twisting like some sort of organic creature.`
+			choice
+				`	Walk over to where she is standing.`
+					goto walkover
+				`	Wait until she notices you.`
+			`	After several minutes of watching Taely commune with the trio of ships, she turns towards you and waves you over. "Enjoying the view?" She signs as you approach, but she continues without waiting for a response. "They are nice ships,and a major step forward in our ship development. Well worth watching." She nods to herself as she runs a hand along a fin, as if checking something.`
+				goto merganserintro
+			label walkover
+			`	Taely waves to you as you approach. "Eager to meet the latest additions to our fleet?" She pats the nose of the nearest ship. "They definitely are a beautiful addition to the Remnant patrols." She smiles to herself and runs a hand along one of the fins as if checking for something.`
+			label merganserintro
+			branch penguin
+				has "event: remnant: penguin"
+			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now, and recent breakthroughs in our materials science have enabled significant progress in our ship designs." She pauses and gestures to one. Would you like to touch it?`
+				goto touch
+			label penguin
+			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch it?`
```

Comment:
I think here I mainly used it instead of them because I had already used them earlier in the paragraph to refer to the group. But yeah, them might sound better.

Reply:
Changed "it" to "them." It does sound better.